rust/codec-jats/src/decode/body.rs

```diff
@@ -130,19 +136,194 @@ fn decode_sec(path: &str, node: &Node, losses: &mut Losses, depth: u8) -> Block
 
 /// Decode a `<title>` to a [`Block::Heading`]
 fn decode_title(path: &str, node: &Node, losses: &mut Losses, depth: u8) -> Block {
-    record_attrs_lost(path, node, [], losses);
-
     let level = node
         .attribute("level")
         .and_then(|level| level.parse::<i64>().ok())
         .unwrap_or(depth as i64);
 
+    record_attrs_lost(path, node, ["level"], losses);
+
     Block::Heading(Heading::new(
         level,
         decode_inlines(path, node.children(), losses),
     ))
 }
 
+/// Decode a `<statement>` to a Stencila [`Block::Claim`]
+///
+/// see https://jats.nlm.nih.gov/archiving/tag-library/1.2/element/statement.html
+fn decode_statement(path: &str, node: &Node, losses: &mut Losses, depth: u8) -> Block {
+    let claim_type = node
+        .attribute("content-type")
+        .map(|statement| ClaimType::from_str(statement).unwrap_or_default())
+        .unwrap_or_default();
+
+    let label = node
+        .children()
+        .find(|child| child.tag_name().name() == "label")
+        .and_then(|node| node.text())
+        .map(String::from);
+
+    let content = decode_blocks(
+        path,
+        node.children()
+            .filter(|child| child.tag_name().name() != "label"),
+        losses,
+        depth,
+    );
+
+    record_attrs_lost(path, node, ["content-type"], losses);
+
+    Block::Claim(Claim {
+        claim_type,
+        label,
+        content,
+        ..Default::default()
+    })
+}
+
+/// Decode a `<figure>` to a Stencila [`Block::Figure`]
+///
+/// see https://jats.nlm.nih.gov/archiving/tag-library/1.2/element/figure.html
+fn decode_figure(path: &str, node: &Node, losses: &mut Losses, depth: u8) -> Block {
+    let mut label_automatically = None;
+    if let Some(automatically) = node
+        .attribute("label-automatically")
+        .map(|string| match string {
+            "true" => Some(true),
+            "false" => Some(false),
+            _ => None,
+        })
+    {
+        label_automatically = automatically;
+    }
```

Comment:
This should be able to be simplified to:

```suggestion
    let label_automatically = node
        .attribute("label-automatically")
        .map(|string| match string {
            "true" => Some(true),
            "false" => Some(false),
            _ => None,
        });
```

or even

```suggestion
    let label_automatically = node
        .attribute("label-automatically")
        .and_then(|string| string.parse().ok());
```

(neither tested)